File: TotalMedia.WebAPI/Models/Country.cs
namespace TotalMedia.WebAPI.Models
{
    internal class Country : IEntity
    {
        internal Country() { }
        
        public Country(byte id, string name, string locale, string currency, List<decimal> vats)
        {
            Id = id;
            Name = name;
            Locale = locale;
            VATs = vats;
            Currency = currency;
        }

        public byte Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Locale { get; set; } = string.Empty;
        public string Currency { get; set; } = string.Empty;
        public List<decimal> VATs { get; set; } = new();
    }
}

File: totalmedia-app/src/app/models/country.ts
export class Country {
  public name: string;
  public vats: number[];
  public locale: string;
  public currency: string;

  constructor(name: string, locale: string, currency: string, vats: number[]) {
    this.name = name;
    this.vats = vats;
    this.locale = locale;
    this.currency = currency;
  }
}
